### ISSUE TITLE
WLCS relies on implicit focus on creation

### ISSUE THREAD

**Issue by vberger** [MirServer/wlcs#240]
It appears some tests (like `CopyCutPaste` for example) rely on the fact that a newly created window would automatically get keyboard focus on creation.

As far as I can tell, this is not something standardized in the protocol and rather this is compositor policy. So I suppose it would make sense to instead add a function to the `wlcs_display_server` interface so that WLCS can explicitly require from the compositor that a certain surface be given keyboard focus? This would typically be used in cases like this `create_surface_with_focus` function:

**Comment by RAOF** [comment]
Yeah, there are a couple of places where we make assumptions about compositor behaviour that aren't strictly required by the protocol (I know there's some stuff about dismissing `xdg_shell` popups, too). We should incrementally addb `wlcs_display_server` interfaces so that those can become explicit.
